lib/movie_main_widget.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'text_styles.dart';
+import 'genres.dart';
+import 'movie.dart';
+import 'movie_title.dart';
+import 'top_row.dart';
+
+class MovieMainWidget extends StatelessWidget {
+  const MovieMainWidget({
+    super.key,
+    required this.movie,
+  });
+
+  final Movie movie;
+
+  static const EdgeInsets topRowPadding = EdgeInsets.only(
+    top: 150,
+    left: 10,
+  );
+
+  static const EdgeInsets movieGenresPadding = EdgeInsets.only(top: 30);
+  static const EdgeInsets movieTitlePadding = EdgeInsets.only(top: 35);
+  static const EdgeInsets movieOverviewPadding = EdgeInsets.only(top: 35);
+  static const double genresSizeBoxHeight = 25;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Stack(
+          children: [
+            Container(
+              alignment: Alignment.topCenter,
+              child: movie.backdrop,
```

Comment:
This should be `Image.network(movie.backdrop)`.
Same below

Reply:
Done